apps/reference/docs/guides/platform/disk-usage.mdx

```diff
@@ -1,45 +1,55 @@
 ---
-id: disk-usage
-title: Disk space usage
-description: Learn how database disk space usage is reported.
+id: database-usage
+title: Database usage
+description: Understanding how database usage applies to your subscription.
 ---
 
-Database disk space usage refers to the _monthly average disk usage_, as reported by Postgres. This metric is reported in your project's [billing page](https://app.supabase.com/project/_/settings/billing) and is updated daily.
+Database storage refers to the _monthly average storage usage_, as reported by Postgres. This metric is reported in your project's [billing page](https://app.supabase.com/project/_/settings/billing) and is updated daily.
 
-For an instantaneous live view of the DB disk space being used by your project, you can execute in Postgres:
+For an instantaneous live view of the DB storage usage, you can execute in Postgres:
 
 ```sql
-SELECT SUM(pg_database_size(pg_database.datname)) / (1024 * 1024) as db_size_mb FROM pg_database;
+select sum(pg_database_size(pg_database.datname)) / (1024 * 1024) as db_size_mb
+from pg_database;
 ```
 
 This value is also reported in the [database settings page](https://app.supabase.com/project/_/settings/database).
 
-## Disk space management
+## Database storage management
 
-Running out of disk space causes the database service to become unresponsive. Operations such as backups and automatic service optimization are also disrupted.
+Your project will be changed to read-only mode once it reaches 95% of your subscrption quota, preventing further writes to your project.
```

Comment:
```suggestion
Your project will be changed to read-only mode once it reaches 95% of your subscription quota, preventing further writes to your project.
```

Looks good! Do we also want to change the doc's filename to `database-usage.mdx` to reflect its new path and id?